### PR TITLE
Upgrade `jquery` to v3 and cleanup related deprecations

### PR DESCRIPTION
**Pull request by shmshd**

### CODE REVIEW

**Comment by evrifaessa** [comment]
Woah, this is big.

**Review by jaapmarcus — APPROVED** [review]

**Review by admdly — COMMENTED** [review]
Looks good to me, apart from query about styling in template.

File: src/modules/Support/html_admin/mod_support_canned_selector.html.twig
Comment (shmshd) [reply src/modules/Support/html_admin/mod_support_canned_selector.html.twig:5]: I was too lazy 😆

**Review by admdly — APPROVED** [review]
Great PR - will merge after leaving for a while for everyone (unless someone beats me to it 😄 ).

File: src/themes/admin_default/package.json
Comment (admdly) [inline src/themes/admin_default/package.json:19]: This package looks to be over 7 years old and unmaintained - should we really be pulling this in?

File: src/themes/admin_default/assets/fossbilling.js
Comment (admdly) [inline src/themes/admin_default/assets/fossbilling.js:5]: This has been depreciated and we shouldn't really be adding it in. Probably best to refactor wherever it is required to avoid using this.

**Review by admdly — CHANGES_REQUESTED** [review]
The latest changes introduce some potential issues with jquery.browser that should be considered/discussed (ideally fixed) before merging.

File: src/themes/admin_default/assets/fossbilling.js
Comment (shmshd) [reply src/themes/admin_default/assets/fossbilling.js:5]: jQuery Alert is an old plugin based on jQuery v1. `jquery.browser` is added as a temporary fix until we replace jQuery Alert.

Comment (shmshd) [reply src/themes/admin_default/assets/fossbilling.js]: @evrifaessa is working on a replacement at PR https://github.com/FOSSBilling/FOSSBilling/pull/903.

Comment (evrifaessa) [reply src/themes/admin_default/assets/fossbilling.js]: I think I won't be able to remove jQuery Alert completely in that PR. I was planning for a gradual migration, then removing it when it's no longer used.

Comment (evrifaessa) [reply src/themes/admin_default/assets/fossbilling.js]: Just checked again. There is only one place left where jAlert is used. jConfirm is cleared. jPrompt is still used in 2 places. I haven't really implemented inputs, forms or anything, so replacing jPrompt will have to wait.

I might not have time to work on replacing jPrompt, so I'd just love to have my PR merged as is before it ends up in conflict hell.

Comment (admdly) [reply src/themes/admin_default/assets/fossbilling.js]: Noted - hopefully we can remove this once the underlying requirement is gone.

File: src/themes/admin_default/package.json
Comment (admdly) [reply src/themes/admin_default/package.json:19]: As with other - can be removed once requirements removed.

**Review by admdly — APPROVED** [review]

**Review by BelleNottelling — APPROVED** [review]
Changes LGTM and this is a huge improvement, thank you!

**Review by evrifaessa — APPROVED** [review]
Haven't done a live installation yet, but this definitely is a great PR and a big improvement. Thanks 🙂

**Review by admdly — APPROVED** [review]
Also not (yet) tested, but changes look good. Happy to merge once tested (likely late today/tomorrow, for me).

**Review by admdly — APPROVED** [review]
Tested and looks to be working as expected, as such merging now.

**Pull request merged**
